packages/orm/src/collections/OneToManyCollection.ts

```diff
@@ -44,13 +42,15 @@ export class OneToManyCollection<T extends Entity, U extends Entity> extends Abs
 
   add(other: U): void {
     ensureNotDeleted(this.entity);
-    if (this.loaded === undefined) {
-      if (!this.addedBeforeLoaded.includes(other)) {
-        this.addedBeforeLoaded.push(other);
+    const loaded = this.loaded;
```

Comment:
Hm. Maybe if you want to get rid of the `this.` prefixes, do this as:

```javascript
const { loaded, addedBeforeLoaded } = this;
```

Reply:
This is actually just leftover changes from my attempts to proxify these functions.  I can revert to the original.

Reply:
Cool, any of reverting or leaving or destructuring is fine.